<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <title>Rithwik - ghar</title>
    <meta name="Description" content="Rithwik's personal website">
    <meta name="viewport" content="width=device-width,initial-scale=1">
    <link rel="icon" type="image/svg+xml" href="heart-solid.svg">
    <link rel="stylesheet" href="/style.css">
  </head>
  <body>
    <div style="display:table;margin:16px auto" class="main-container">
      <div class="c">
        <pre style="top:0;position:sticky;user-select:none;padding:16px;box-sizing:border-box">


<h1 class="big">Rithwik Narendra</h1>         
<a href="/">Home</a>

          
<a href="/About-me">+ about me</a>

<a href="/Bio-Inf-Work">+ Bio Inf Work</a>
<a href="/Publications">+ publications</a>        

<a href="/Other-projects">+ Other Projects</a>
<a href="/Writing">+ writing</a>

<a href="/(Not-so)-important-links">+ (Not so) Important Links</a>
<a href="/Contact">+ contact</a>      
<a href="https://github.com/rukja/">+ GitHub</a>
        </pre>
      </div>
      <div class="c">
        
        <pre>
+------------------------------------------------------------------------------+
|                                                                              |
|                              Bioinformatics Work                             |
|                                                                              |
+------------------------------------------------------------------------------+

These are projects to which I have contributed substantially. There are projects 
I have worked on which are not listed here. To learn more about those, visit the 
<a href="/Publications">Publications</a> link on the left. 
________________________________________________________________________________

Project: Computationally assessing <i>Mycobacterium abscessus</i> phage susceptibility
Group: Farhat Lab @ Harvard Department of Biomedical Informatics
Dates: June '25 - Ongoing
Stage: Ongoing

Motivation: The rise of "superbugs," or bacterium that are resistant to
antibiotics poses a significant threat. <i>Mycobacterium asbcessus</i>, or 
M. abscessus, a nontuberculous mycobacterium, is one of these. In recent years,
M. abscessus cases have substantially increased, as has its ability to become 
resistant to many treatments. Bacteriophages are highly-specific viruses which 
can infect and kill bacteria. Phage therapy for M. abscessus is promising, but 
due to specificity of phages, it is difficult to ascertain whether a given phage
will infect a bacteria strain. Bioinformatics can play a role here.

My work: Utilizing bioinformatics to annotate, characterize, and extract features 
from phage and M. abscessus strain genomes. Additionally, leveraging protein 
language models to summarize the protein landscape of phages and strains. 
Developing a machine learning model to predict infection based on these features. 
Additionally, building a state-space genomic language model to process protein 
and intergenic embeddings to predict infection and discover biology via 
explainability mechanisms. 

________________________________________________________________________________

________________________________________________________________________________

Project: Developing a response expression quantitative trait loci pipeline
Group: Wells Lab @ UCLA David Geffen School of Medicine
Dates: August '24 - Ongoing
Stage: Ongoing

Motivation: Humans have widespread genetic variation. This genetic variation 
can have implications in the developing brain. Expression quantitative loci 
(EQTLs) are variants which lead to differing levels of gene expression. Some 
variants lead to differing levels of gene expression when the cell is exposed to 
an environmental condition, such as viral infection. Broadly, these variants are 
known as response EQTLs. Understanding response EQTLs in the context of 
neurodevelopment is incredibly important to assess the impact of potential 
environmental exposures on the developing brain. 

My work: Building a pipeline which utilizes linear regression and stringent
false discovery measures to determine the relationship between genotype 
variants, environmental exposures, and gene expression from neural progenitor 
cells. Pipeline includes extensive pre-processing and quality control, and 
robust checks for variable redundancy and ancestral stratification. 

________________________________________________________________________________
           
________________________________________________________________________________

Project: Building a parsimonious classifier to predict COVID-19 trajectory
Group: Langelier Lab @ UCSF Division of Infectious Disease
Dates: June '24 - Ongoing
Stage: Second-stage of review at JCI Insight, <a href = "https://www.medrxiv.org/content/10.1101/2025.05.18.25327658v1" target = "_blank">Preprint</a>

Motivation: The immense challenges hospitals faced in appropriately allocating
resources during the COVID-19 pandemic partially stemmed from difficulties
in accurately triaging incoming positive patients. A prognostic classifier
based on patient information within 48-hours of hospital admission could
assist in this process while providing insights into the biology of COVID-19
mortality and potential information about hetereogeneity of treatment in 
response to drug treatment.

My work: Analyzing the differences in gene expression and proteomic data 
between varying COVID-19 trajectories. Comparing differntial pathway and gene
expression with Sepsis mortality trajectories. Developing a LASSO regression
classifier which utilizes a minimal set of genes to predict COVID-19 mortality
with the addition of age and cycle threshold metrics. Validating classifier
in external datasets which include vaccinated patients to assess 
generalizability. Comparing classifier performance with existing published 
classifiers and "gold-standard" hospital assessments. 

________________________________________________________________________________  
           
________________________________________________________________________________

Project: A robust co-expression pipeline for single cell data
Group: Wells Lab @ UCLA David Geffen School of Medicine
Dates: October '23 - August '24
Stage: Used by lab

Motivation: The developing brain exhibits numerous phenotypes, and these may 
change in response to treatment. These phenotypes may be correlated to genes,
but can also be correlated with groups, or modules, of genes that are 
co-expressed. Understanding how co-expressed genes in the developing brain
associate with phenotypes such as cell growth and division and exhibition of 
patterning factors and associate with treatments such as valproic acid or lead 
is crucial for enhancing understanding of early neurobiology and what genes
could provide a protective effect for harmful exposures. 

My work: Building a <a href = "https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-9-559" target = "_blank">WGCNA</a> pipeline to process single cell neural progenitor gene expression. Enable
the user to select the appropriate initial parameters before utilizing a robust 
resampling method to ensure "module stability." Conduct pathway, hub gene, and 
differential eigengene expression analysis to characterize the biology of 
modules and their relationships with traits and treatments. Most steps of the 
pipeline can be completed as job submissions. 

________________________________________________________________________________ 

________________________________________________________________________________

Project: Investigating inflammaging in indiviudals with Lupus
Group: Langelier Lab @ UCSF Division of Infectious Disease
Dates: June '23 - May '25
Stage: Published, <a href = "https://www.science.org/doi/10.1126/scitranslmed.adt5550" target = "_blank">Science Translational Medicine</a>

Featured in: <a href = "https://www.ucsf.edu/news/2025/07/430391/how-aging-quiets-lupus-and-brings-relief-some-older-patients" target = "_blank">UCSF News</a>

Motivation: Aging in healthy human populations is associated with increases in 
inflammation, which can lead to chronic disease. In Lupus, aging is associated
with improvements with respect to symptoms and outcomes. Investigating this 
apparent contradiction would increase our understanding of the biology of aging,
Lupus, and provide us with cellular mechanisms to potentially combat 
age-associated inflammation. 

My work: Conducted bulk gene expression analysis to identify differentially
expressed genes with age in patients with Lupus and in controls. Analyzed 
pathways using gene set enrichment analysis to explore the relationship 
between immune-associated pathways and age. Performed similar analysis
at the single-cell level to assess which specific cell types were implicated in 
this differential expression with age. Additionally assessed changes in 
cell type proportion with respect to age in both groups. Investigated 
relationship between epigenetic changes (primarily methylation) to quantify 
the relationship between age and interferon-related genes in patients with Lupus.
Performed proteomic analysis to identify proteins differentially 
associated with age in Lupus patients, and determine the concentration of these 
proteins over time.

________________________________________________________________________________ 

Rithwik Narendra (C) 2025-
        </pre>
      </div>
    </div>
  </body>
</html>
